apps/rush-lib/assets/rush-init/common/config/rush/command-line.json

```diff
@@ -118,7 +118,12 @@
        *
        * For details, refer to the website article "Using watch mode".
        */
-      "watchForChanges": false
+      "watchForChanges": false,
+
+      /**
```

Comment:
If the build cache is still experimental, add `(EXPERIMENTAL)` for this setting.